.eslintcache

```diff
@@ -1 +1 @@
n":9,"nodeType":"58","endLine":26,"endColumn":37},{"ruleId":"52","severity":1,"message":"61","line":11,"column":10,"nodeType":"54","messageId":"55","endLine":11,"endColumn":23},{"ruleId":"52","severity":1,"message":"62","line":12,"column":10,"nodeType":"54","messageId":"55","endLine":12,"endColumn":28},{"ruleId":"52","severity":1,"message":"63","line":13,"column":10,"nodeType":"54","messageId":"55","endLine":13,"endColumn":29},{"ruleId":"52","severity":1,"message":"64","line":14,"column":10,"nodeType":"54","messageId":"55","endLine":14,"endColumn":20},{"ruleId":"52","severity":1,"message":"65","line":15,"column":10,"nodeType":"54","messageId":"55","endLine":15,"endColumn":17},{"ruleId":"52","severity":1,"message":"66","line":16,"column":10,"nodeType":"54","messageId":"55","endLine":16,"endColumn":26},{"ruleId":"52","severity":1,"message":"67","line":17,"column":10,"nodeType":"54","messageId":"55","endLine":17,"endColumn":26},{"ruleId":"52","severity":1,"message":"68","line":18,"column":10,"nodeType":"54","messageId":"55","endLine":18,"endColumn":32},{"ruleId":"52","severity":1,"message":"69","line":19,"column":10,"nodeType":"54","messageId":"55","endLine":19,"endColumn":26},{"ruleId":"52","severity":1,"message":"70","line":20,"column":10,"nodeType":"54","messageId":"55","endLine":20,"endColumn":22},{"ruleId":"52","severity":1,"message":"71","line":21,"column":10,"nodeType":"54","messageId":"55","endLine":21,"endColumn":20},{"ruleId":"52","severity":1,"message":"72","line":24,"column":10,"nodeType":"54","messageId":"55","endLine":24,"endColumn":22},{"ruleId":"52","severity":1,"message":"73","line":27,"column":10,"nodeType":"54","messageId":"55","endLine":27,"endColumn":20},{"ruleId":"52","severity":1,"message":"74","line":28,"column":10,"nodeType":"54","messageId":"55","endLine":28,"endColumn":26},{"ruleId":"52","severity":1,"message":"75","line":29,"column":10,"nodeType":"54","messageId":"55","endLine":29,"endColumn":22},"no-unused-vars","'individualUserData' is defined but never used.","Identifier","unusedVar","jsx-a11y/alt-text","img elements must have an alt prop, either with meaningful text, or an empty string for decorative images.","JSXOpeningElement","jsx-a11y/heading-has-content","Headings must have content and the content must be accessible by a screen reader.","'getAllRecipes' is defined but never used.","'getUserWithRecipes' is defined but never used.","'boughtRecipesByUser' is defined but never used.","'recipeById' is defined but never used.","'getUser' is defined but never used.","'createIngredient' is defined but never used.","'createUserRecipe' is defined but never used.","'updateUserRecipeRating' is defined but never used.","'searchNonProfits' is defined but never used.","'registerUser' is defined but never used.","'userSignIn' is defined but never used.","'createRecipe' is defined but never used.","'updateUser' is defined but never used.","'updateIngredient' is defined but never used.","'updateRecipe' is defined but never used."]
```

Comment:
Idk what this is lol

Reply:
I was just talking about this is the student channel, it doesn't seem to be immportant or easy to gitignore with the most recent react script 🤷🏻‍♂️ 
I wouldn't worry about it.

Reply:
I only committed this file because I thought it might be important?